audio/espeakedit/Portfile

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           makefile 1.0
+
+name                espeakedit
+version             1.48.03
+revision            0
+categories          audio
+platforms           darwin
+maintainers         nomaintainer
+license             GPL-3+
+
+description         eSpeakEdit is used to prepare and compile phoneme data for the eSpeak speech synthesier.
+
+
+long_description    eSpeakEdit is used to prepare and compile phoneme data for the eSpeak speech synthesier. \
+                    In addition to its own functions, espeakedit is compiled including the source files from the  speak  program. \
+                    So it uses the same synthesis routines when it produced sound from phoneme data.
+
+homepage            http://espeak.sourceforge.net/
+set branch          [join [lrange [split ${version} .] 0 1] .]
+master_sites        sourceforge:project/espeak/espeak/espeak-${branch}
+distname            ${name}-${version}
+use_zip             yes
+checksums           rmd160  7e5306c473a8f6ab3128726f1693561929fb315c \
+                    sha256  9142e029b2cf8ceac8592bd2845ed34c38525172c17fe9cfe885e7b3ee3e1a75 \
+                    size    7557985
+pre-patch {
+    copy -force ${build.dir}/portaudio19.h ${build.dir}/portaudio.h
+}
+
+patchfiles          patch-espeakedit-mac.diff
+
+depends_lib         port:wxgtk-2.8
+depends_lib-append  port:espeak
+
+if {${os.platform} eq "darwin" && ${os.major} >= 10} {
+    depends_lib-append      port:portaudio
+    configure.args-append   --with-mictype=portaudio
+}
+
+compiler.cxx_standard 1998
+configure.cxxflags-append -std=c++98
+build.dir           ${worksrcpath}/src
+makefile.override   PREFIX
+build.args          CC=${configure.cc} \
+                    CXX=${configure.cxx} \
+                    CPP=${configure.cpp}
```

Comment:
Why are these build args needed? The makefile 1.0 portgroup is supposed to take care of it for you.